exchanges/gdax/gdax.go

```diff
@@ -237,14 +237,28 @@ func (g *GDAX) GetHistoricRates(currencyPair string, start, end, granularity int
 	}
 
 	for _, single := range resp {
+		var s History
+		a, _ := single[0].(float64)
+		s.Time = int64(a)
+		b, _ := single[1].(float64)
+		s.Low = b
+		c, _ := single[2].(float64)
+		s.High = c
+		d, _ := single[3].(float64)
+		s.Open = d
+		e, _ := single[4].(float64)
+		s.Close = e
+		f, _ := single[5].(float64)
+		s.Volume = f
+		/* type assertion poosible here, since sometimes gdax returns empty value
 		s := History{
```

Comment:
This code block can be removed before merge.

Reply:
shall i make another commit to remove the commented out code

Reply:
Yes please, thanks for all the good work!